content/en/docs/managing-jx/common-tasks/tls.md

```diff
@@ -6,4 +6,19 @@ date: 2019-08-28
 weight: 190
 ---
 
-Please see the [boot ingress documentation](/docs/getting-started/setup/boot/#ingress)
\ No newline at end of file
+As documented in the [boot ingress documentation](/docs/getting-started/setup/boot/#ingress) , you can configure a global default certificate for all jenkins x ingresses. this required having dns enable for a wildcard certificate. in case this not possible and you need to generate one certificate per ingress, you can specify the `secreName`s to be used in the values.yaml for each ingress for example:
```

Comment:
Would you mind splitting one sentence per line. That makes it easier to make further changes.

Capitalization seems a bit off. Here is what I would suggest:

> As documented in the [boot ingress section](/docs/getting-started/setup/boot/#ingress), you can configure a global default certificate for all Jenkins X ingresses.
> This requires having DNS enabled for a wildcard certificate. 
> In case this not possible and you need to generate one certificate per ingress, you can specify the `secretName` to be used in the `values.yaml` for each ingress for example:

Reply:
Technically, I am not quite sure what this means. Would it make sense to explain this a bit more?

Reply:
I tried to improve the technical part and fix the capitalization, could you review again please?

thank you :)